tests/sys/psa_crypto_cipher/Makefile

```diff
@@ -6,8 +6,9 @@ USEMODULE += ztimer_usec
 USEMODULE += psa_crypto
 
 USEMODULE += psa_cipher
+USEMODULE += psa_cipher_chacha_20
```

Comment:
naming nitpicking again: the keytype is called `CHACHA20` (without the underscore), would you mind adapting all the module/periph feature mentions accordingly?

Reply:
ok, i'll rename it